Reports/progress_report.md

```diff
@@ -0,0 +1,292 @@
+# Data Safe Haven project report
+
+This document contains a summary of progress across all stories in the [project roadmap](https://github.com/orgs/alan-turing-institute/projects/111/views/1).
+It maps stories according to the (main) pillar and priority they contribute to.
+
+Included stories are those that were scheduled and/or prioritised over this period (February 2023-February 2024), some not scheduled stories are included when there are updates for them regardless (indirect contributions from other stories, relevant work recently started, or something to report in general).
+
+## Infrastructure as code
```

Comment:
I don't think all of this section is "Infrastructure as Code".
Is that the pillar name? I think I might have made this argument before. It is probably more fair to call it "codebase development" or something like that.